gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +0,0 @@
-#Thu Jul 19 15:03:45 MSK 2018
```

Comment:
Any gradle project must have the files in `/gradle/wrapper` have repeatable builds. Please revert those files back.

Reply:
reverted. Thanks